website/src/components/CustomerLogos/index.tsx

```diff
@@ -4,85 +4,93 @@ import Marquee from "react-fast-marquee";
 
 export function CustomerLogosGrayscale() {
   return (
-    <div className="mx-8 grid grid-cols-3 md:grid-cols-7 gap-8">
-      <div className="content-center">
-        <Link href="https://corrdyn.com">
-          <Image
-            alt="corrdyn logo"
-            src="/images/cust-logo-corrdyn-gray.svg"
-            width={125}
-            height={125}
-            className="opacity-75 md:opacity-25 mx-auto"
-          />
-        </Link>
-      </div>
-      <div className="content-center">
-        <Link href="https://square1.io">
-          <Image
-            alt="square1 logo"
-            src="/images/cust-logo-square1-gray.svg"
-            width={100}
-            height={100}
-            className="opacity-75 md:opacity-50 mx-auto"
-          />
-        </Link>
-      </div>
-      <div className="content-center">
-        <Link href="https://bunq.com">
-          <Image
-            alt="bunq logo"
-            src="/images/cust-logo-bunq-gray.svg"
-            width={100}
-            height={100}
-            className="opacity-75 mx-auto"
-          />
-        </Link>
-      </div>
-      <div className="content-center">
-        <Link href="https://wolfram.com">
-          <Image
-            alt="wolfram logo"
-            src="/images/cust-logo-wolfram-gray.svg"
-            width={75}
-            height={75}
-            className="opacity-75 md:opacity-100 mx-auto"
-          />
-        </Link>
-      </div>
-      <div className="content-center">
-        <Link href="https://sebgroup.com">
-          <Image
-            alt="seb logo"
-            src="/images/cust-logo-seb-gray.svg"
-            width={75}
-            height={75}
-            className="opacity-75 mx-auto"
-          />
-        </Link>
-      </div>
-      <div className="content-center">
-        <Link href="https://teracloud.com">
-          <Image
-            alt="teracloud logo"
-            src="/images/cust-logo-teracloud-gray.svg"
-            width={125}
-            height={125}
-            className="opacity-75 md:opacity-50 mx-auto"
-          />
-        </Link>
-      </div>
-      <div className="content-center col-start-2 md:col-auto">
-        <Link href="https://double11.co.uk">
-          <Image
-            alt="double11 logo"
-            src="/images/cust-logo-double11-gray.svg"
-            width={70}
-            height={70}
-            className="opacity-75 md:opacity-25 mx-auto"
-          />
-        </Link>
+    <section
+      className="flex w-full h-full max-w-full max-h-full items-center justify-center m-0 p-0 list-none opacity-100 overflow-hidden"
+      style={{
+        maskImage:
+          "linear-gradient(to right, rgba(0, 0, 0, 0) 0%, rgb(0, 0, 0) 35%, rgb(0, 0, 0) 65%, rgba(0, 0, 0, 0) 100%)",
+      }}
+    >
```

Comment:
This can be simplified to the below, which has the same effect

```suggestion
    <div
      style={{
        maskImage:
          "linear-gradient(to right, rgba(0, 0, 0, 0) 0%, rgb(0, 0, 0) 35%, rgb(0, 0, 0) 65%, rgba(0, 0, 0, 0) 100%)",
      }}
    >
```

Also the `<section>` should be `<div>` since it's just used for styling and isn't a new "section" of the page.